_config.yml

```diff
@@ -6,7 +6,7 @@ author: Josh Davis
 description:
     List of sites with Two Factor Auth support which includes SMS, email,
     phone calls, hardware, and software.
-url: http://twofactorauth.org
+url: https://twofactorauth.org
```

Comment:
We have to use HTTP on this line as it otherwise won't let users with incompatible browsers access our site without HTTPS.